### PR TITLE
[IA-2235] Bump RStudio version

### PR DESCRIPTION
**Pull request by rtitle**
See: 
https://broadworkbench.atlassian.net/browse/IA-2235
https://github.com/anvilproject/anvil-docker/pull/20
https://github.com/DataBiosphere/leonardo/pull/1674

I'm also bumping it in Leo automation tests and generating new custom GCE/Dataproc images. My plan is to make this live next week after the next Leo release.

@gpcarr how do I generate a new `terra-docker-versions-new.json` after merging this?

### CODE REVIEW

**Comment by gpcarr** [comment]
were both the new base image and bioconductor image built and pushed to the anvil GCR?

**Comment by rtitle** [comment]
Yeah -- `anvil-rstudio-base` is pushed, `anvil-rstudio-bioconductor` build is in progress. I'll push when it's done (should be soon).

**Comment by gpcarr** [comment]
> Yeah -- `anvil-rstudio-base` is pushed, `anvil-rstudio-bioconductor` build is in progress. I'll push when it's done (should be soon).

Sounds good - code LGTM once bioconductor is pushed

**Review by gpcarr — APPROVED** [review]

**Comment by rtitle** [comment]
Pushed now :) 

@gpcarr do you know how I generate a new `terra-docker-versions-new.json` after merging this?

**Review by Qi77Qi — APPROVED** [review]
lgtm

**Comment by gpcarr** [comment]
> Pushed now :)
> 
> @gpcarr do you know how I generate a new `terra-docker-versions-new.json` after merging this? (Or will it happen automatically?)

sorry I missed this earlier! it should happen automatically however i do see a bunch of aborted builds in https://fc-jenkins.dsp-techops.broadinstitute.org/view/Leonardo/job/leonardo-build-terra-docker/. I also see that the `terra-docker-versions-new.json` file in the google bucket has the latest `anvil-rstudio-bioconductor` version in it. it's possible the job is detecting whether or not any of the terra maintained images were updated and aborting if none have. 

running the https://github.com/DataBiosphere/terra-docker/blob/master/scripts/generate_version_docs.py manually will both generate a new `terra-docker-versions-new.json` and push it to the google bucket, is that what you ended up doing?

**Pull request merged**
